frontend/express/public/javascripts/countly/countly.views.js

```diff
@@ -2611,6 +2611,53 @@ window.EventsBlueprintView = countlyView.extend({
             },[jQuery.i18n.map["events.general.cancel"],jQuery.i18n.map['events.general.confirm']]);
         });
         
+        var segments = [];
+        for(var i=0; i<self.activeEvent.segments.length; i++)
+        {
+            segments.push({"key":self.activeEvent.segments[i],"value":self.activeEvent.segments[i]});
+        }
+        for(var i=0; i<self.activeEvent.omittedSegments.length; i++)
+        {
+            segments.push({"key":self.activeEvent.omittedSegments[i],"value":self.activeEvent.omittedSegments[i]});
+        }
+        
+        $('#event-management-projection').selectize({
```

Comment:
Selectize must be added as core javascript library, including adding in Gruntfile.js and dashboard.html